Add Running as a Docker Container section (#1)
spiceai/cookbook: docker/README.md

> # Running in Docker Reciple

Reviewer: ```suggestion
# Running in Docker Recipe
```

> # Quickstart: Running Spice as a Docker Container

Reviewer: ```suggestion
# Running in Docker Reciple
```

> This reciple demonstrates how to run Spice.ai OSS in a container. The recipe uses [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html) run as a container with Spice using Docker Compose configuration and Open AI model to perform vector search and natural language queries to the dataset.

Reviewer: ```suggestion
This recipe demonstrates how to run Spice.ai OSS in a container. The recipe uses [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html) run as a container with Spice using Docker Compose configuration and Open AI model to perform vector search and natural language queries to the dataset.
```

> Quickstart demonstrates how to run Spice.ai OSS as a container. The quickstart uses [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html) run as a container with Spice using Docker Compose configuration and Open AI model to perform vector search and natural language queries to the dataset.

Reviewer: ```suggestion
This reciple demonstrates how to run Spice.ai OSS in a container. The recipe uses [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html) run as a container with Spice using Docker Compose configuration and Open AI model to perform vector search and natural language queries to the dataset.
```

> Quickstart demonstrates how to run Spice.ai OSS as a container. The quickstart uses [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html) run as a container with Spice using Docker Compose configuration and Open AI model to perform vector search and natural language queries to the dataset.

Reviewer: We already ship a Docker container - so its a little confusing why we're building one here. I think it should clarify that this is for baking in the spicepod.yaml, otherwise you can just use the pre-built docker images

Author: Update to:

This recipe demonstrates how to run Spice.ai OSS in a container using the official [spiceai/spiceai](https://hub.docker.com/r/spiceai/spiceai) Docker image. It shows how to:

- Extend the [spiceai/spiceai](https://hub.docker.com/r/spiceai/spiceai) Docker image with a desired configuration (`spicepod.yml`, secrets, etc).
- Use Docker Compose to run Spice locally with a sample dataset containerized - [MySQL Sakila Sample Database](https://dev.mysql.com/doc/sakila/en/sakila-structure-tables.html).
- Interact with containerized Spice runtime using Spice CLI tools
